src/styles/blocks/form.scss

```
@@ -0,0 +1,34 @@
+.form {
+
+  &-input {
+    display: block;
+    box-sizing: border-box;
+    height: 48px;
+    width: 100%;
+    padding: 14px 16px;
+    font-family: "Cinzel", serif;
+    border: 1px solid transparent;
+    background-color: #f8f8fa;
+    transition: border-color 0.3s;
+
+    &::placeholder {
+      font-size: 14px;
+      line-height: 20px;
+    }
+
+    &:hover {
+      border-color: #d5d7de;
```

```suggestion
      border-color: $some-var;
```

Use variables for colors, usually used styles property and etc.